ledger/participant-integration-api/src/main/scala/platform/store/dao/LedgerDao.scala

```diff
@@ -128,17 +112,9 @@ private[platform] trait LedgerReadDao extends ReportsHealth {
 
   def transactionsReader: LedgerDaoTransactionsReader
 
-  def completions: LedgerDaoCommandCompletionsReader
+  def contractsReader: LedgerDaoContractsReader
```

Comment:
I thought we agreed that from this working track we will do minimal impact change to the LedgerDao, meaning in this particular case: only adding necessary methods to LedgerDao.
Also for later: I would be more happier if we would align first on how we wan't to refactor/segregate/make LedgerDao nicer, so all efforts are coordinated and fit to a final goal this regard.
Let's discuss